app/src/components/integrations/integrationProviders/ldapIntegration/constants.js

```diff
@@ -26,12 +26,19 @@ export const PASSWORD_ATTRIBUTE_KEY = 'passwordAttribute';
 export const URL_KEY = 'url';
 export const BASE_DN_KEY = 'baseDn';
 export const EMAIL_KEY = 'email';
+export const NAME_TYPE_KEY = 'nameType';
 export const FULL_NAME_KEY = 'fullName';
+export const FIRST_AND_LAST_NAME_KEY = 'firstAndLastName';
+export const FIRST_NAME_KEY = 'firstName';
+export const LAST_NAME_KEY = 'lastName';
 export const PHOTO_KEY = 'photo';
 export const LDAP_PREFIX = 'ldap://';
 export const LDAPS_PREFIX = 'ldaps://';
 
+export const FULL_NAME = 'FULL_NAME';
```

Comment:
Is it an alternative for `FIRST_AND_LAST_NAME_KEY`?
Can we align they naming if yes?